### PR TITLE
Post on ELIXIR exchange project

### PR DESCRIPTION
**Pull request by MarisaJL**

### CODE REVIEW

**Comment by MarisaJL** [comment]
Just need to add link to the tutorial when it is available as it's currently waiting to be reviewed

File: content/blog/2024-11-elixir-exchange/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Wrapping Seurat for Galaxy
+
+During the exchange I learned how tools are incorporated into Galaxy through wrappers, which set out the tool requirements, the options that users will see on the interface, and the commands that Galaxy will perform when someone hits the Run Tool button. I used what I learned to wrap the latest version of the Seurat pipeline, which I was already familiar with thanks to my focus on single cell omics during my PhD. 
+
+I drew on my experience of using Galaxy to reorganise the Seurat tools, grouping them together by theme, so that we now have wrappers for preprocessing, dimensional reduction, data integration, and other activities. Each of these wrappers contains a selection of related functions, including some that weren’t previously available on Galaxy. Once the tools were working, I also wrote a tutorial for the Galaxy Training Network (GTN) to help people start using them for their own analyses. 
+
+### The New Seurat Toolsuite
+
+- **Seurat Create:** load transcriptomic or CITEseq data into a Seurat Object, add QC metrics, and filter the dataset
```
Comment (bgruening) [inline content/blog/2024-11-elixir-exchange/index.md:23]: You can add links to the tools if you like or even buttons like here https://github.com/galaxyproject/galaxy-hub/blob/e54dc35cd16ef7accc4218dd41bedb8cd7c2fa78/content/news/2022-02-24-alphafold/index.md?plain=1#L46

**Comment by bgruening** [comment]
@MarisaJL thanks a lot. Let me know when you think this is ready.

**Comment by MarisaJL** [comment]
Hi @bgruening, this is ready now - I don't have the tutorial link yet as that's waiting to be reviewed, but I want to get the blog up now so I can share it!

**Comment by bgruening** [comment]
Then let's ship it. You can add the link later.

**Comment by bgruening** [comment]
@MarisaJL can I move this into the news section? Of have you used the link already in some report?

**Comment by MarisaJL** [comment]
> @MarisaJL can I move this into the news section? Of have you used the link already in some report?

I already used it and sent the report earlier!!

**Pull request merged**